joomla-projects/gsoc18_override_management
# Quick icon notification and Test

Pull Request for Issue #22 

### Summary of Changes
Quick Icon notification in control panel


### Testing Instructions
Try to update any extension. And make sure you have created  override of that extension before update.

### Expected result
![screenshot from 2018-06-25 16-06-25](https://user-images.githubusercontent.com/30978328/41845772-4321d706-7892-11e8-84f0-d23e23e71855.png)
![screenshot from 2018-06-25 16-06-38](https://user-images.githubusercontent.com/30978328/41845776-47bd6712-7892-11e8-880f-cc535ae5db73.png)

NOTE: Some of the code is written only for test and they removed after the test.
